releasenotes/notes/refactor-sbom-parameters-4af241e77fc00f83.yaml

```diff
@@ -0,0 +1,116 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Refactor the SBOM collection parameters from::
+
+      conf.d/container_lifecycle.d/conf.yaml existence (A) # to schedule the container lifecycle long running check
+      conf.d/container_image.d/conf.yaml     existence (B) # to schedule the container image metadata long running check
+      conf.d/sbom.d/conf.yaml                existence (C) # to schedule the sbom long running check
+
+      Inside datadog.yaml:
+
+      container_lifecycle:
+        enabled:                        (D)  # Used to control the start of the container_lifecycle forwarder but has been decommissioned by #16084 (7.45.0-rc)
+        dd_url:                              # \
+        additional_endpoints:                # |
+        use_compression:                     # |
+        compression_level:                   #  > generic parameters for the generic EVP pipeline
+          …                                  # |
+        use_v2_api:                          # /
+
+      container_image:
+        enabled:                        (E)  # Used to control the start of the container_image forwarder but has been decommissioned by #16084 (7.45.0-rc)
+        dd_url:                              # \
+        additional_endpoints:                # |
+        use_compression:                     # |
+        compression_level:                   #  > generic parameters for the generic EVP pipeline
+          …                                  # |
+        use_v2_api:                          # /
+
+      sbom:
+        enabled:                        (F)  # control host SBOM collection and do *not* control container related sbom since #16084 (7.45.0-rc)
```

Comment:
```suggestion
        enabled:                        (F)  # control host SBOM collection and do **not** control container-related SBOM since #16084 (7.45.0-rc)
```